lib/console.c

```diff
@@ -48,14 +49,13 @@ static bool is_listed (const char *cfgin, const char *tty, bool def)
 
 	if (*cons != '/') {
 		char *pbuf;
+
 		STRTCPY(buf, cons);
-		pbuf = &buf[0];
-		while ((s = strtok (pbuf, ":")) != NULL) {
+		pbuf = buf;
+		while (NULL != (s = strsep(&pbuf, ":"))) {
```

Comment:
Can we maintain the NULL comparison at the end? I know we have some parts of the code where it's used at the beginning, but at least for me it's easier to read the comparison having it at the end.

Reply:
I agree with you that within conditionals it's usually much easier to read `if (p == NULL)` than `if (NULL == p)`.

However, when an assignment is done within the conditional, it's easy to make mistakes, and often results in bugs.  The only way to avoid that is moving the constant expression to the left-hand side.  That's pretty much the only place and reason that I reverse the expression.